.gitmodules

```diff
@@ -10,3 +10,7 @@
 	path = micro-ros-stm32_embeddedrtps/micro-ros_embeddedrtps/micro_ros_stm32cubemx_utils
 	url = https://github.com/mROS-base/micro_ros_stm32cubemx_utils
 	branch = galactic
+[submodule "mros2-posix"]
+	path = mros2-posix
+	url = git@github.com:mROS-base/mros2-posix
+	branch = eval/v0.1.0
```

Comment:
README and directory seem to be `eval/v0.2.0` (differ to this). My suggestion is to create `eval/v0.2.0` branch on mros2-posix repository for consistency.

Reply:
I assumed that `v0.2.0` in README.md and the directory name corresponds to the version of `eval` repo (and the merged commit will be tagged `v0.2.0`). Does it make sense?

Reply:
Oops,  you're right! sorry for my lack of memory,,,